Makefile.am

```diff
@@ -60,12 +60,36 @@ AM_CPPFLAGS = \
 # Utility library
 noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
 
+enum_header_files = \
+	libeos-updater-util/flatpak.h \
+	eos-updater-flatpak-installer/installer.h \
+	$(NULL)
+enum_template_h_file = libeos-updater-util/enums.h.in
+enum_template_c_file = libeos-updater-util/enums.c.in
+enum_sources = \
+	libeos-updater-util/enums.h \
+	libeos-updater-util/enums.c \
+	$(NULL)
+
+# FIXME: We can't use the --output switch as documented in https://git.gnome.org/browse/glib/commit/?id=cd97f93bf7c44cb5736f6cbcb4db773ca592e462
```

Comment:
Ah, it was because `--output` didn’t work if the target file didn’t exist.

I’ve merged your patch for that to endlessm/glib, so it should be OK to use here now. Thanks.